core/tests/protractor.conf.js

```diff
@@ -319,9 +323,52 @@ exports.config = {
     // will be available. For example, you can add a Jasmine reporter with:
     //     jasmine.getEnv().addReporter(new jasmine.JUnitXmlReporter(
     //         'outputdir/', true, true));
-    var _ADD_SCREENSHOT_REPORTER = true;
 
-    if (_ADD_SCREENSHOT_REPORTER) {
+    var spw = '';
+    var uniqueString = '';
+    var vidPath = '';
+    // Enable allVideos if you want success videos to be saved.
+    var allVideos = false;
+
+    // Only running video recorder on Github Actions.
+
+    if (process.env.PWD == '/home/runner/work/oppia/oppia') {
+      jasmine.getEnv().addReporter({
+        specStarted: function(){
+          let ffmpegArgs = [
+            '-y',
+            '-r', '30',
+            '-f', 'x11grab',
+            '-s', '1280x1024',
+            '-i', process.env.DISPLAY,
+            '-g', '300',
+            '-loglevel', '16',
```

Comment:
What does this do since we don't have handlers for STDOUT and STDERR anymore?

Reply:
oops, must have deleted them by accident. Added them back in